src/styles/blocks/page.scss

```diff
@@ -0,0 +1,24 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+
+  .container {
+    box-sizing: border-box;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 50px 0;
+    width: 944px;
+  }
+
+  .card {
```

Comment:
instead of using chained css selectors add a BEM mix for example for `card` block `main__card`
and use it here in the following way:

```
.main {
  &__card {
    // external styles
  }
}
```